### PR TITLE
Create script to run gene selection from expression matrices for running on CodeOcean.

### PR DESCRIPTION
**Pull request by morriscb**

### CODE REVIEW

File: scripts/create_gene_expression.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    if args.species == "human":
+        directory_name = "WHB-10Xv3"
+    elif args.species == "mouse":
+        directory_name = "WMB-10X"
+    else:
+        raise ValueError(f"Unknown species requested: {args.species}")
+
+    cell = abc_cache.get_metadata_dataframe(
+        directory='WMB-10X',
```
Comment (danielsf) [inline scripts/create_gene_expression.py:77]: Should this be `directory=directory_name`?

**Review by danielsf — CHANGES_REQUESTED** [review]
I think there's just one typo where you've left it hard-coded for mouse

File: scripts/create_gene_expression.py (hunk 1 above)
Comment (morriscb) [reply scripts/create_gene_expression.py:77]: Good catch. That's what I get for copy/pasting from a notebook.

**Review by danielsf — APPROVED** [review]
lgtm

**Pull request merged**